test/test_jit.py

```diff
@@ -8288,17 +8288,10 @@ def as_interface(x):
                    .check("Traceback") \
                    .check("in foo").check("in baz").run(str(cm.exception))
 
-    def test_binop_unsupported_error(self):
-        with self.assertRaisesRegex(NotSupportedError, "unsupported binary operator:"):
-            @torch.jit.script
-            def binop(x, y):
-                # Replace this with another unsupported op when/if it gets supported
-                return x << y
-
     def test_bitwise_ops(self):
```

Comment:
Could you also add some test that uses a bunch of bitwise / binary / unary operators to have some check on the operator precedence?

Reply:
Added a test for operator precedence